kits/democracy/contracts/DemocracyKit.sol

```diff
@@ -0,0 +1,63 @@
+pragma solidity 0.4.24;
+
+import "@aragon/kits-beta-base/contracts/BetaKitBase.sol";
+
+
+contract DemocracyKit is BetaKitBase {
+    constructor(
+        DAOFactory _fac,
+        ENS _ens,
+        MiniMeTokenFactory _minimeFac,
+        IFIFSResolvingRegistrar _aragonID,
+        bytes32[4] _appIds
+    )
+        BetaKitBase(_fac, _ens, _minimeFac, _aragonID, _appIds)
+        public
+    {}
+
+    function newToken(string name, string symbol) external returns (MiniMeToken token) {
+        token = minimeFac.createCloneToken(
+            MiniMeToken(address(0)),
+            0,
+            name,
+            18,
+            symbol,
+            true
+        );
+        cacheToken(token, msg.sender);
+    }
+
+    function newInstance(
+        string name,
+        address[] holders,
+        uint256[] tokens,
+        uint256 supportNeeded,
+        uint256 minAcceptanceQuorum,
+        uint64 voteDuration
+    )
+        external
+    {
+        MiniMeToken token = popTokenCache(msg.sender);
+        Kernel dao = createDAO(
+            name,
+            token,
+            holders,
+            tokens,
+            uint256(-1)
+        );
+
+        Voting voting = Voting(dao.getApp(dao.APP_ADDR_NAMESPACE(), appIds[uint8(Apps.Voting)]));
+        voting.initialize(
+            token,
+            supportNeeded,
+            minAcceptanceQuorum,
+            voteDuration
+        );
+
+        // Burn support modification permission
+        ACL acl = ACL(dao.acl());
```

Comment:
If we return return the Voting app as @sohkai proposes, maybe we can also return the `ACL` so we save a call.

Reply:
But if we return `ACL`, `Voting`, and `TokenManager`, what do we want `Kernel` for? I'm afraid I'm confused.

Reply:
In the case `Democracy` and `Multisig` they don't need a kernel reference as all the apps have already been deployed (but other future kits that use `beta-base` might), but as @sohkai said it feels weird that `createDAO` doesn't return a reference to the main DAO contract.

I think for consistency we could return the Kernel, ACL and the 4 apps that were deployed, even if our kits only need some of them.

Reply:
Originally I was thinking about the shotgun approach and just returning everything but didn't want to bloat the return size too much.

But if the alternative is to need calls to get the same info anyway, then we might as well return some extra data.

Reply:
Returning more than 3 objects was throwing the following error: `CompilerError: Stack too deep, try removing local variables.`